content/docs/lists-and-keys.md

```diff
@@ -237,18 +237,18 @@ function Blog(props) {
 }
 
 const posts = [
-  {id: 1, title: 'Hello World', content: 'Welcome to learning React!'},
-  {id: 2, title: 'Installation', content: 'You can install React from npm.'}
+  {id: 1, title: 'Olá Mundo', content: 'Bem vindo para aprender React!'},
+  {id: 2, title: 'Instalação', content: 'Você pode instalar a React utilizando o npm.'}
```

Comment:
```suggestion
  {id: 2, title: 'Instalação', content: 'Você pode instalar o React utilizando o npm.'}
```

Reply:
It's something that we need to think about because React is a library = biblioteca, and using `o` for me seems odd, what do you think about that?

Reply:
Responderei em português: 98% dos artigos aqui no Brasil e vídeos tutoriais falam: o React.
Exemplo: https://tableless.com.br/react-o-que-e-e-como-funciona-essa-ferramenta/

Reply:
What do you think about it @cezaraugusto ? For me I prefer `o React`